src/bench/bench.h

```diff
@@ -49,22 +59,24 @@ struct Args {
     fs::path output_csv;
     fs::path output_json;
     std::string regex_filter;
+    uint8_t priority_level;
```

Comment:
nit: I think `priority` better highlights the difference between the aggregated priority and the individual priority level
```suggestion
    uint8_t priority;
```